corehq/apps/dhis2/tasks.py

```diff
@@ -52,4 +53,4 @@ def send_datavalues(domain_name):
 )
 def send_datasets_for_all_domains():
     for domain_name in find_domains_with_toggle_enabled(toggles.DHIS2_INTEGRATION):
-        send_datavalues(domain_name)
+        send_datasets.apply(args=(domain_name,))
```

Comment:
Do you want this to run synchronously? If so you can still just call the function normally, if not you should use `.delay`